lib/airbrake-ruby/filter_chain.rb

```diff
@@ -67,7 +68,8 @@ def root_directory_filter(root_directory)
       proc do |notice|
         notice[:errors].each do |error|
           error[:backtrace].each do |frame|
-            frame[:file].sub!(/\A#{root_directory}/, '[PROJECT_ROOT]'.freeze)
+            next unless (file = frame[:file])
+            file.sub!(/\A#{root_directory}/, '[PROJECT_ROOT]'.freeze)
           end
```

Comment:
Also you probably should combine those 2 filters together to save some CPU time.